### PR TITLE
Made Changes in README and Contributing sections

### PR DESCRIPTION
**Pull request by aflah02**
ISSUE #50 

fixes #

I've fixed some grammatical errors and moved the contribution related section to CONTRIBUTING file

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**
NA

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by kaiwalyakoparkar — COMMENTED** [review]
LGTM, I think there a PR which is going to add an additional command to npm section. This merge should go good after that merge

**Comment by aflah02** [comment]
@kaiwalyakoparkar 
Alright then thanks!

**Comment by aflah02** [comment]
@kaiwalyakoparkar there seems to be a conflict again it seems this is because other requests were merged before mine. What should I do now?

**Comment by kaiwalyakoparkar** [comment]
Check what content was added recently. This may occur because your code contains some line of code which are changed in the newer update.  Hence the current readme code is conflicting with your older readme code. Try to figure it out. If you couldn't then I will suggest you an alternate way.

**Comment by aflah02** [comment]
@kaiwalyakoparkar I tried to last time as well and this time too. I clicked on Resolve Conflicts but can't seem to understand what went wrong. Any help would be very appreciated :)

**Comment by kaiwalyakoparkar** [comment]
Firstly save your this changes somewhere. And then try pulling the upstream code. This will update the recent code on your locale machine and then check the main branch and branch with your changes. Hope you get the answer

**Comment by aflah02** [comment]
@kaiwalyakoparkar got it should i do this rn or first wait for that other request npm one to be merged? Has it been merged already?

**Comment by kaiwalyakoparkar** [comment]
Yes it is merged. I think the same might have created an conflict. The change are in the same section which you are shifting. You can see new command of npm install . Which is not a part of your readme

File: CONTRIBUTING.md
Comment (kaiwalyakoparkar) [inline CONTRIBUTING.md:83]: The new `npm install` command is added here in the new readme. Which is not a part of your readme

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md
Comment (aflah02) [reply CONTRIBUTING.md:83]: alright i'll move this to the contributing section too

**Comment by kaiwalyakoparkar** [comment]
I think you just added that line to contributing.md that won't work. You will have to pull this readme file and then cut the content from it and paste it in contributing.md

**Comment by aflah02** [comment]
@kaiwalyakoparkar I'll do that then i guess

**Comment by aflah02** [comment]
I've generated a new pull request @kaiwalyakoparkar so I'm closing this

**Comment by kaiwalyakoparkar** [comment]
@aflah02 No problem could you please link that PR here so that I can review it

**Comment by aflah02** [comment]
Here you go @kaiwalyakoparkar https://github.com/codeforcauseorg/edu-client/pull/94

**Pull request closed without merging**
